etl/command.py

```diff
@@ -89,6 +94,13 @@ def run_dag(
     if not include_grapher:
         excludes.append("grapher://")
 
+    _validate_private_steps(dag)
+
+    if not private:
+        # NOTE: could this possibly go to `excludes`?
+        dag = _filter_private_steps(dag)
+        print("Excluding private steps...")
```

Comment:
I think your intuition is right, `private://` could be appended to the excludes list here to get the same behaviour.